src/main/java/com/hotels/hcommon/hive/metastore/core/MetaStoreClientException.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (C) 2017-2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.hcommon.hive.metastore.core;
```

Comment:
I'm thinking we get rid of the "core" package as it doesn't really make sense here now that we're outside of Circus Train.